packages/providers/README.md

```diff
@@ -17,57 +17,24 @@ but for those with more specific needs, individual components can be imported.
 
 ```javascript
 const {
-
     Provider,
     BaseProvider,
-
-    JsonRpcProvider,
-    StaticJsonRpcProvider,
-    UrlJsonRpcProvider,
-
-    FallbackProvider,
-
-    AlchemyProvider,
-    CloudflareProvider,
-    EtherscanProvider,
-    InfuraProvider,
-    NodesmithProvider,
-
-    IpcProvider,
-
-    Web3Provider,
-
-    WebSocketProvider,
-
-    JsonRpcSigner,
-
+    DefaultHederaProvider,
+    HederaProvider,
     getDefaultProvider,
-
     getNetwork,
-
+    composeHederaTimestamp,
```

Comment:
I don't think that we need to exoport this function. Can we change it so that it is not exported?